pkgs/by-name/pa/patch2pr/package.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, testers
+, patch2pr
+}:
+
+buildGoModule (self: {
```

```suggestion
buildGoModule rec {
```

buildGoModule does not recognize finalAttrs design pattern yet.